doc/build/dts/intro-syntax-structure.rst

```diff
@@ -67,6 +69,12 @@ Devicetree nodes can also have *properties*. Properties are name/value pairs.
 Property values can be any sequence of bytes. In some cases, the values are an
 array of what are called *cells*. A cell is just a 32-bit unsigned integer.
 
+.. _dt-property-labels:
+
+Just like nodes, properties can also have *property labels*, which use the same
```

Comment:
I left this out of this document on purpose, because to my knowledge we have no uses for property labels in zephyr.

The specification as usual explains this detail, but I don't understand why we need it in this zephyr-specific document. Can you please explain?